agate/pattern-virtualgridlist-api/src/components/ImageItem/ImageItem.js

```diff
@@ -1,11 +1,11 @@
+import ImageItem from '@enact/agate/ImageItem';
 import kind from '@enact/core/kind';
-import GridListImageItem from '@enact/moonstone/GridListImageItem';
 import PropTypes from 'prop-types';
 import {connect} from 'react-redux';
 
 import {selectItem} from '../../actions';
 
-const ImageItem = kind({
+const ImageItems = kind({
```

Comment:
How about GalleryItem?
ImageItems looks set of ImageItem.
```suggestion
const GalleryItem = kind({
```

Reply:
It's fixed.